docs/content/index.md

```diff
@@ -62,7 +62,7 @@ These environments can be extended, version controlled, and shared, so you can t
     
     1. Install Docker with [recommended settings](users/install/docker-installation.md#windows).
     2. Install [DDEV for Windows](users/install/ddev-installation.md#windows).
-    3. Launch your [first project](users/project) and start developing. 🚀
+    3. Launch your [first project](users/project.md) and start developing. 🚀
```

Comment:
I wonder why these weren't caught by links check.

Reply:
I saw the warning in the build only after I fixed the deprecated emoji plugins 🙂.
The link itself works as before (mkdocs gives us some slack).